product_extended_segmentation/wizard/wizard_price.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models
+
+
+class WizardPrice(models.Model):
+    _inherit = "wizard.price"
+
+    def onchange_recursive(self, cr, uid, ids, recursive, context=None):
```

Comment:
When triggering onchange we have to be aware of the active model
a lá https://github.com/Vauxoo/addons-vauxoo/blob/8.0/product_extended_variants/wizard/wizard_price.py#L45

This away with are pointing to the right model

This is because wizard can be called from model product.product or product.template

Regards.